lib/IRGen/GenReflection.cpp

```diff
@@ -1451,6 +1517,7 @@ void IRGenModule::emitFieldDescriptor(const NominalTypeDecl *D) {
     if (strategy.getElementsWithPayload().size() > 1 &&
         !strategy.needsPayloadSizeInMetadata()) {
       needsOpaqueDescriptor = true;
+      needsMPEDescriptor = true;
```

Comment:
TODO?  Is this the right place to suppress MPE descriptors for unbound generic enums?

Reply:
After some more inspection, I think this is actually correct already.